code/script.js

```diff
@@ -2,6 +2,22 @@
 const board = document.getElementById('board')
 const questions = document.getElementById('questions')
 const restartButton = document.getElementById('restart')
+const findOutButton = document.getElementById('filter')
+const winOrLose = document.getElementById('winOrLose');
```

Comment:
I see that you used semicolon a bit inconsistently, I would advice to choose one side (for or against semicolons in JS) and stick to it.